README.md

```diff
@@ -19,12 +19,31 @@ Include the package locally in your repository.
 
 `npm install dotenv-webpack --save`
 
+### Description
+
+`dotenv-webpack` wraps `dotenv` and `Webpack.DefinePlugin`. As such, it overwrites existing anyexisting `DefinePlugin` configurations. Also, like `DefinePlugin`, it does a text replace in the resulting bundle for any instances of `process.env`. 
+
+As such, if used with `React` in production, make sure to include `NODE_ENV=production` inside your `.env` file.
```

Comment:
@mrsteele I'm removing this text and adding an example 
###### Use in your code

```
// file1.js
console.log(process.env.DB_HOST);
// '127.0.0.1'
```

###### Resulting bundle
```
// bundle.js
console.log({
	DB_HOST='127.0.0.1',
	DB_PASS='foobar',
	S3_API='mysecretkey'
}.DB_HOST);
```